tournaments.json

```
@@ -232,6 +232,28 @@
                 "Marcel",
                 "Mukhtar"
             ]
+        },   
+        {
+            "date": "2025-02-23",
```

The game was yesterday ;)
```suggestion
            "date": "2025-02-22",
```